src/app/satellite.ts

```diff
@@ -5,17 +5,22 @@ export class Satellite {
 	launchDate: string;
 	orbitType: string;
 	operational: boolean;
-
+	warning: boolean;
 	constructor(name: string, type: string, launchDate: string, orbitType: string, operational: boolean) {
 		this.name = name;
 		this.type = type;
 		this.launchDate = launchDate;
 		this.orbitType = orbitType;
 		this.operational = operational;
+		this.warning = this.isSpaceDebris();
    }
 	
 	isSpaceDebris(): boolean {
-		return true;
+		if(this.type === "Space Debris"){
```

Comment:
you can shorten this by just saying `return this.type === "Space Debris"`. This will already evaluate as true or false, and return the value of the operator. No need for the if/else check.